pkgs/applications/blockchains/framesh/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, fetchurl, appimageTools, ... }:
+
+let
+  pname = "framesh";
+  version = "0.5.0-beta.19";
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://github.com/floating/frame/releases/download/v${version}/Frame-${version}.AppImage";
+    sha256 = "sha256-/mmVnUnJVfoKfXMN2zPlo+lsUninCNuENmVA2x8dDEY=";
+  };
+
+  appimageContents = appimageTools.extractType2 {
+    inherit name src;
```

Comment:
I don't think `name` is necessary here at all.

Reply:
``appimageTools.extractType2 {} `` requires either name or pname and version.

Reply:
All right. Not too familiar with AppImage stuff myself.